server/src/main/java/com/broadcom/lsp/cobol/core/semantics/GroupContext.java

```diff
@@ -0,0 +1,83 @@
+package com.broadcom.lsp.cobol.core.semantics;
+
+import com.broadcom.lsp.cobol.core.messages.MessageService;
+import com.broadcom.lsp.cobol.core.model.ErrorSeverity;
+import com.broadcom.lsp.cobol.core.model.Locality;
+import com.broadcom.lsp.cobol.core.model.SyntaxError;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import lombok.NoArgsConstructor;
+import org.eclipse.lsp4j.Location;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor
+public class GroupContext {
```

Comment:
Pleae, provide javadocs for the class and all the public methods

Reply:
Done